src/v2/availability.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import Http from 'wrecked';
+import route, { Kind } from './shared/routes';
+
+
+export default (apiKey, baseUrl) => {
+
+  const http = Http.create();
+
+  const getRoute = (kind, ...params) => route('availability', kind, { baseUrl, apiKey, ...params });
+
+  return {
+
+    all(pmcId, unitId) {
+      const uri = getRoute(Kind.COLLECTION, pmcId, unitId);
+      return http.get(uri);
+    },
+
+    checkByDate(pmcId, unitId) {
+      const uri = getRoute(Kind.CHECK, pmcId, unitId);
+      return http.get(uri);
+    },
+
+    replace(pmcId, unitId, payload) {
+      const uri = getRoute(Kind.COLLECTION, pmcId, unitId);
+      return http.put(uri, payload);
+    },
+
+    delete(pmcId, unitId) {
+      const uri = getRoute(Kind.COLLECTION, pmcId, unitId);
+      return http.delete(uri);
+    },
+
+    status(pmcId, unitId, workflowId) {
+      const uri = getRoute(Kind.WORKFLOW, pmcId, unitId, workflowId);
```

Comment:
I saw that. I guess I didn't explain what I meant properly. In that file, the Routes object has an availability object. That availability object does not have workflow property.